pkg/actor/decommission.go

```diff
@@ -98,7 +97,7 @@ func (d decommission) Act(ctx context.Context, cluster *resource.Cluster) error
 		log.Error(err, "We cannot decommission if there are less than 3 nodes", "nodes", nodes)
 		return err
 	}
-	log.V(int(zapcore.InfoLevel)).Info("replicas decommisioning", "status.CurrentReplicas", status.CurrentReplicas, "expected", cluster.Spec().Nodes)
+	log.Info("replicas decommisioning", "status.CurrentReplicas", status.CurrentReplicas, "expected", cluster.Spec().Nodes)
```

Comment:
why is this line a different format than 88 or 114?

Reply:
Because you can call `log.Info` directly and it takes into consideration the logging level already.  You do not have to set the logging level by hand.